### PR TITLE
function calls can include integers and test script 

### PR DESCRIPTION
**Pull request by kennethgoodman**
test script looks at the input and expected output and checks the difference.

### CODE REVIEW

File: tests/run_tests.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import os
+from rockstarpy.rockstar import convert_code
+import difflib
+from pprint import pprint
+try:
+    from StringIO import StringIO # python 2
```
Comment (deepserket) [inline tests/run_tests.py]: Quoting setup.py
`'Programming Language :: Python :: 3 :: Only'`

File: tests/run_tests.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import os
+from rockstarpy.rockstar import convert_code
```
Comment (deepserket) [inline tests/run_tests.py]: PEP8 Imports

File: tests/run_tests.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        for d in diff:
+            print(diff)
+        assert False, "There are differences"
+
+def main():
+    files = os.listdir('.')
+    rock_files = filter(lambda f: '.rock' == f[-5:], files)
+    py_files = set(filter(lambda f: '.py' == f[-3:], files))
+    for rock_file in rock_files:
+        file_name = rock_file[:-5]
```
Comment (deepserket) [inline tests/run_tests.py]: you are assuming that all rockstar files uses ".rock" extension, but on the help for the argument "input" there are listed three different extensions `(rock|rockstar|lyrics)`.
I think that `rock_file.rstrip('.', 1)[0]` is better

File: tests/run_tests.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+def main():
+    files = os.listdir('.')
+    rock_files = filter(lambda f: f.split(".")[-1] in ['rock','rockstar','lyrics'] , files)
+    py_files = set(filter(lambda f: '.py' == f[-3:], files))
+    for rock_file in rock_files:
+        file_name = rock_file[:-5]
+        py_file = file_name + ".py"
+        assert py_file in py_files, "Did not create a corrosponding expected output for " + file_name + ".rock"
+
```
Comment (deepserket) [inline tests/run_tests.py]: Good, there is some code that can be improved:

line 24: `py_files = set(filter(lambda f: f.endswith('.py'), files))`
line 26: `file_name = rock_file.rstrip('.', 1)[0]`
line 28: `assert py_file in py_files, "Did not create a corrosponding expected output for " + rock_file`

**Comment by yanorestes** [comment]
Thank you! I'm accepting the PR. I'm planning to add support to Python 2, so soon I'll get back your try/except.

**Pull request merged**